src/cholesky.jl

```diff
@@ -65,7 +65,7 @@ julia> C.U
   ⋅    ⋅    3.0
 
 julia> C.L
-3×3 LowerTriangular{Float64, Adjoint{Matrix{Float64}}}:
+3×3 LowerTriangular{Float64, {Matrix{Float64}}:
```

Comment:
This doesn't look right, or?  Maybe

```suggestion
3×3 LowerTriangular{Float64, Matrix{Float64}}:
```

Reply:
There must be a mismatch between used code and tests in the failing doctest. This has been changed deliberately lately IIRC.

Reply:
My guess is that the docs don't run with the custom sysimage and are in fact testing the bundled LinearAlgebra. Could that be correct?